Dockerize button handler

aitorcas23 opened the pull request:
Dockerize button handler by generating an image using a Dockerfile and running it with docker compose.

Some characteristics of the docker service:
- Is in privileged mode.
- Runs in a custom docker network called iombian-internal-services.

aitorcas23 committed:
Add firts try of Dockerfile

aitorcas23 committed:
Docker build and run working

aitorcas23 committed:
Use environment variables

aitorcas23 committed:
Update Dockerfile and create docker-compose.yml

aitorcas23 committed:
Add docker to readme.md

The docker-compose stuff is still missing.

aitorcas23 committed:
Working without network mode host

bodiroga commented on README.md:
````diff
@@ -51,6 +51,42 @@ Any ZeroMQ subscriber can listen to those events and act acordingly.
+
+After building the image, execute it with docker run
+
+```docker run --network host -e BUTTON_PIN=3 --privileged --rm -d --name ${CONTAINER_NAME} ${IMAGE_NAME}```
````
```docker run --name ${CONTAINER_NAME} --privileged --rm -d -p 5556:5556 -e BUTTON_PIN=3 ${IMAGE_NAME}:${IMAGE_VERSION}```

I would prefer if the official documentation did not recommend the use of "--network host" and suggested the mapping of the port to use: 5556.

bodiroga reviewed (changes requested):
Awesome work @aitorcas23, many thanks for your help! 😉

aitorcas23 committed:
Expose button events port

Expose button events port.
Now the internal button events port can't be changed, only the external port.

aitorcas23 committed:
Get button events port from environment variable

aitorcas23 committed:
Docker compose metadata labels

aitorcas23 committed:
Remevo docker-compose.yaml

aitorcas23 committed:
Fix docker section of README

aitorcas23 committed:
Spelling mistakes

bodiroga reviewed (approved):
LGTM! 👍🏻
